<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMain</class>
 <widget class="QWidget" name="WindowMain">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>868</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Employee Database Interface</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>851</width>
     <height>581</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab:selected 
{
    color: white;
    background: rgb(245, 102, 37); 
border: 1px solid black;
   border-radius: 5px;
padding: 0 10;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_my_edi_2">
    <attribute name="title">
     <string>My EDI</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Team</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Projects</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>280</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 2px dotted rgb(245, 103, 37);
border-radius: 15px;</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>Shift Details</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>Start Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>End Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>dd/mm/yyyy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string>hh:mm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string>hh:mm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QCalendar" name="calendarWidget1_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>304</width>
       <height>173</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Projects</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>My Schedule</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LogOut_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>10</y>
       <width>112</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_control_2">
    <attribute name="title">
     <string>Control</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_searchEmployee">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>0</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ID">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_FirstName">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_LastName">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Salary">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_SSN">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SSN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Position">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Username">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SearchEmployee">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Employee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_vieweditemployee_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>View/Edit Employee</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>841</width>
       <height>461</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>823</width>
        <height>459</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="5">
        <widget class="QTableWidget" name="tableWidget_db_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>500</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>EmployeeID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>First Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Salary</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SSN</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Username</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Password</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Team</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Company Employees</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="5">
        <widget class="QTableWidget" name="tableWidget_db_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>199</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Team ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Team Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Projects Assigned</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Projects Completed</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButton_newTeam">
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>New Team</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="lineEdit_searchTeam">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Teams</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_SearchTeam">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Search Team</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="pushButton_viewEditTeam">
         <property name="text">
          <string>View/Edit Team</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_TeamReset">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_addEmployee_13">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>New Employee</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Team">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Team</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Reset">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_analyze_2">
    <attribute name="title">
     <string>Analyze</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/new/prefix1/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelEDI">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EMPLOYEE DATABASE INTERFACE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LogOut">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_welcomeUser">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Logout_image">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/new/prefix2/EDI-LogOut-Button.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCalendar</class>
   <extends>QCalendarWidget</extends>
   <header>qcalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
